.travis.sh

```diff
@@ -51,6 +51,7 @@ for test_l in irteus/test/*.l; do
 
     # osrf/ubuntu_arm64:trusty takes >50 min, skip irteus-demo.l
     [[ "$DOCKER_IMAGE" == *"arm64:trusty"* && $test_l =~ irteus-demo.l ]] && continue;
+    [[ ( "$DOCKER_IMAGE" == *"trusty"* || "$DOCKER_IMAGE" == *"jessie"* ) && $test_l =~ test-collision.l ]] && continue;
```

Comment:
please add comment why we have to skip this test.

Reply:
added comment: https://github.com/euslisp/jskeus/pull/555/commits/1f4da2f7da05f3106eea44366a1cfc92cd07277f#diff-b4553af39acae935d505c76e139d4aa3R54-R55
(rebase commit into previous)